ai-data/managed-inference/reference-content/mistral-memo-instruct-2407.mdx

```diff
@@ -0,0 +1,82 @@
+---
+meta:
+  title: Understanding the Mistral-nemo-instruct-2407 model
+  description: Deploy your own secure Mistral-nemo-instruct-2407 model with Scaleway Managed Inference. Privacy-focused, fully managed.
+content:
+  h1:  Understanding the Mistral-nemo-instruct-2407 model
+  paragraph: This page provides information on the Mistral-nemo-instruct-2407 model
+tags: 
+dates:
+  validation: 2024-08-20
+categories:
+  - ai-data
+---
+
+## Model overview
+
+| Attribute       | Details                            |
+|-----------------|------------------------------------|
+| Provider        | [Mistral](https://mistral.ai/technology/#models)                         |
+| Model Name      | `mistral-nemo-instruct-2407`       |
+| Compatible Instances | H100 (FP8)                 |
+| Context size | 128K tokens    |
+
+## Model name
+
+```bash
+mistral-nemo-instruct-2407:fp8
+```
+
+## Compatible Instance
+
+- [H100 (FP8)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+
+## Model introduction
+
+Mistral Nemo is a state-of-the-art transformer model of 12B parameters, built by Mistral in collaboration with NVIDIA.
+This model is open-weight and distributed under the Apache 2.0 license.
+It was trained on a large proportion of multilingual and code data.
+
+## Why is it useful?
+
+- Mistral Nemo offers a very large context window of up to 128k tokens, particularly useful for RAG applications. 
+- It is easy to use and a drop-in replacement in any system already using Mistral 7B.
+- This model was designed for global, multilingual applications. It is particularly strong in English, French, German, Spanish, Italian, Portuguese, Chinese, Japanese, Korean, Arabic, and Hindi. 
+
+## How to use it
+
+### Sending Inference requests
+
+<Message type="tip">
+  Unlike previous Mistral models, Mistral Nemo requires smaller temperatures. It is recommend to use a temperature of 0.35.
+</Message>
```

Comment:
Maybe specify the unit of temprature?

Reply:
There is no unit in case of ML https://www.iguazio.com/glossary/llm-temperature/